File: fixtures/scale/dart/small/lib/style/styles.dart
const _esc = 'ESC[';
const _reset = '${_esc}0m';

String bold(String text) => '${_esc}1m$text$_reset';

String dim(String text) => '${_esc}2m$text$_reset';

String underline(String text) => '${_esc}4m$text$_reset';

String colorize(String text, int ansiCode) => '$_esc${ansiCode}m$text$_reset';

String strip(String text) => text.replaceAll(RegExp(r'ESC\[[0-9;]*m'), '');

File: fixtures/scale/scala/small/src/textprinter/style/Styles.scala
package textprinter.style

object Styles {
  private val Esc = "\u001b["
  private val Reset = Esc + "0m"

  def bold(text: String): String = Esc + "1m" + text + Reset

  def dim(text: String): String = Esc + "2m" + text + Reset

  def underline(text: String): String = Esc + "4m" + text + Reset

  def colorize(text: String, ansiCode: Int): String =
    Esc + ansiCode.toString + "m" + text + Reset
}
